luaunit.lua

```diff
@@ -175,6 +175,22 @@ local function sortedPairs(tbl)
 end
 M.private.sortedPairs = sortedPairs
 
+math.randomseed(os.clock()*100000000000)
+
+local function randomizeTable( t )
+    -- Shuffle the table t
+    local rand = math.random 
+    assert( t, "shuffleTable() expected a table, got nil" )
```

Comment:
1. I guess that you have probably changed the function name - but it now mismatches the assertion, which says "shuffleTable()".
2. If you want a table type, then test for it - `assert(t)` may be insufficient:

```lua
    if type(t) ~= "table" then
        error("randomizeTable() expected a table, got " .. type(t))
    end
```